tools/depends/target/python3/darwin_embedded.patch

```diff
@@ -231,11 +232,11 @@
 +#  undef HAVE_WAIT
 +#  undef HAVE_WAIT3
 +#  undef HAVE_WAIT4
-+#  undef HAVE_WAITPID
++//#  undef HAVE_WAITPID
  
  /*[clinic input]
  # one of the few times we lie about this name!
-@@ -475,6 +489,10 @@
+@@ -472,6 +486,10 @@
```

Comment:
Is the current range information incorrect?

Reply:
Yes